Source/Krypton Components/Krypton.Toolkit/General/OutlookGrid/CustomColumns/RatingCell.cs

```diff
@@ -79,12 +79,12 @@ public RatingCell()
         /// <param name="cellStyle"></param>
         /// <param name="advancedBorderStyle"></param>
         /// <param name="paintParts"></param>
-        protected override void Paint(Graphics graphics, Rectangle clipBounds, Rectangle cellBounds, int rowIndex, DataGridViewElementStates elementState, object value, object formattedValue, string errorText, DataGridViewCellStyle cellStyle, DataGridViewAdvancedBorderStyle advancedBorderStyle, DataGridViewPaintParts paintParts)
+        protected override void Paint(Graphics graphics, Rectangle clipBounds, Rectangle cellBounds, int rowIndex, DataGridViewElementStates elementState, object? value, object? formattedValue, string? errorText, DataGridViewCellStyle cellStyle, DataGridViewAdvancedBorderStyle advancedBorderStyle, DataGridViewPaintParts paintParts)
         {
-            Image? cellImage = (Image)formattedValue;
+            Image? cellImage = formattedValue as Image;
             if (!ReadOnly)
             {
-                int starNumber = GetStarFromMouse(cellBounds, DataGridView.PointToClient(Control.MousePosition));
+                int starNumber = GetStarFromMouse(cellBounds, DataGridView!.PointToClient(Control.MousePosition));
```

Comment:
seems like `DataGridView` is always being nullable ignored, so change it's type maybe ?

Reply:
public class DataGridViewImageCell : DataGridViewCell
The DataGridView reference is inherited from System.Windows.Forms.DataGridViewImageCell